grunt-tasks/concat.js

```diff
@@ -25,15 +25,15 @@ module.exports = function(grunt) {
         "<%= jsPath %>/lib/bugform.js",
         "<%= jsPath %>/templates.js"
       ],
-      dest: "<%= jsPath %>/<%= pkg.name %>.js"
+      dest: "<%= jsPath %>/dist/<%= pkg.name %>.js"
```

Comment:
Maybe we could make a new Gruntfile variable`jsDistPath` in Gruntfile.js?

Reply:
I can do that, sure.